index.html

```diff
@@ -172,18 +172,21 @@ <h2>Contact me</h2>
 
             </div>
             <div class="contact-items">
-                <form action="https://formspree.io/f/xeqwnkae" method="post" class="form-flex">
+                <form action="https://formspree.io/f/xeqwnkae" method="post" class="form-flex" id="contactForm">
                     <div class="form-flex-item ">
                         <input type="text" placeholder="Lorem Ipsum" maxlength="30" name="userName">
                     </div>
                     <div class="form-flex-item ">
-                        <input type="emial" placeholder="loremipsum@gmail.com" name="emial">
+                        <input type="emial" placeholder="loremipsum@gmail.com" name="email" id="email">
```

Comment:
- [ ] You did well implementing the email validation, :+1:  But I think there is a bug in your code and you need to look into this before moving to the next milestone.
- - Currently, the input field does not have the correct type and therefore it does not check if the users' input is an email or not, This is due to a typo error in your input type, I'll suggest you change the email input type from `type="emial"` to `type="email"`. :smile: 
- - Looking at the video below, you can see that a user can submit the form without adding an email address, This is not great as we need the email to make validations and also it will be used eventually by you to reach out to anyone who wants to contact you, I'll suggest you add the `required` attribute to the email input tag to fix this problem. :smile: :smile:  :wave: 

- - Video for reference:

https://github.com/Salarhabibi-amir/Salarhabibi-amir.github.io/assets/107632780/2e32cf0a-741e-4333-9591-5ca657d31a4d